quarkusQS/src/test/java/my/groupId/QuarkusTestConstants.java

```diff
@@ -0,0 +1,6 @@
+package my.groupId;
+
+public class QuarkusTestConstants {
+    public static final String APP_NAME = "${azure.appName:quarkus-helloWorld}";
```

Comment:
In what scenario are those default values valid? There is timestamp as a suffix.

Reply:
They will never be valid, but I was thinking it would be nice to quickly see in what format name is, of course It would become useless with change in pom, should I remove it altogether?

Reply:
OK, lets keep it. It may serve as a good example.